### ISSUE TITLE
[v2] `aws configure list-profiles` errors when an invalid profile is configured, despite not relying on that information

### ISSUE THREAD

**Issue by benkehoe** [aws/aws-cli#5546]
- [x] I've gone though the [User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) and the [API reference](https://docs.aws.amazon.com/cli/latest/reference/)
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-cli/issues) and didn't find any solution
  
**Describe the bug**
`aws configure list-profiles` should not be affected by invalid session configuration, like an invalid profile, because it doesn't actually use the profile information. However, because it creates a session to read the config, an invalid profile causes errors.

**SDK version number**
aws-cli/2.0.48 Python/3.7.3 Linux/4.4.0-18362-Microsoft exe/x86_64.ubuntu.18

**To Reproduce (observed behavior)**
```bash
$ export AWS_PROFILE=invalid_profile
$ aws configure list-profiles
The config profile (invalid_profile) could not be found
```
No additional information is printed with `--debug`

This can't be worked around by explicitly providing `--profile` because of https://github.com/aws/aws-cli/issues/5016

**Expected behavior**
It should not be affected by invalid session configuration.

**Comment by kdaily** [comment]
Thanks @benkehoe and for linking to #5016.

**Comment by kdaily** [comment]
Hi @benkehoe, I confirmed that this has been fixed by #6066, available in AWS CLI version 2.1.36. Thanks again!
